### PR TITLE
Widen frontend_server_client constraint

### PR DESCRIPTION
**Pull request by sigurdm**
To unblock: https://github.com/dart-lang/pub/pull/3514

### CODE REVIEW

File: pkgs/test_core/pubspec.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ dependencies:
   analyzer: '>=3.3.0 <5.0.0'
   async: ^2.5.0
   args: ^2.0.0
-  boolean_selector: ^2.1.0
+  boolean_selector: '>=2.1.0 <4.0.0'
```
Comment (jakemac53) [inline pkgs/test_core/pubspec.yaml:13]: It looks like 2.1.0 is the latest version here, (also this isn't frontend_server_client?)

**Comment by jakemac53** [comment]
I think this can probably be closed, I did already widen the constraint, although we are figuring out if/how we are going to publish given we have a constraint on a pre-release SDK currently.

**Comment by natebosch** [comment]
A version of `test` is published with a widened constraint.

File: pkgs/test_core/pubspec.yaml (hunk 1 above)
Comment (sigurdm) [reply pkgs/test_core/pubspec.yaml:13]: 🤦 thanks

**Pull request closed without merging**